### PR TITLE
Update iconography and country symbol foundation docs

### PR DESCRIPTION
**Pull request by mark-tate**

### CODE REVIEW

**Comment by mark-tate** [comment]
@amritadesmet how do we feel about splitting into two pages ?

**Review by amritadesmet — DISMISSED** [review]

**Comment by amritadesmet** [comment]
@mark-tate Happy with splitting this - looks good to me now!

File: site/src/components/country-symbol-preview/CountrySymbolPreview.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            No country symbols found for the search term: "
+            <strong>{deferredSearch}</strong>"
+          </Text>
+        </StackLayout>
+      </StackLayout>
+    );
+  }, [filteredSymbols, deferredSearch]);
+
+  return (
+    <Suspense fallback="Loading...">
```
Comment (origami-z) [inline site/src/components/country-symbol-preview/CountrySymbolPreview.tsx:78]: Apart from fixing lint, this takes quite long to load, do you want a different / better loading message ? e.g. using the spinner component?

File: site/src/components/country-symbol-preview/CountrySymbolPreview.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        <FlowLayout justify="start" gap={1}>
+          {filteredSymbols.map(({ countryCode, countryName }) => (
+            <StackLayout
+              style={{ width: 140 }}
+              gap={1}
+              align="center"
+              key={countryCode}
+            >
+              <LazyCountrySymbol code={countryCode} size={1} />
+              <Text>{countryCode}</Text>
```
Comment (origami-z) [inline site/src/components/country-symbol-preview/CountrySymbolPreview.tsx:54]: Since looking, do you feel this needs to be bolder?

**Review by amritadesmet — APPROVED** [review]

File: site/docs/foundations/iconography/country-symbols.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+title: Country symbols
+layout: DetailTechnical
+---
+
+Country symbols represent a specific country and are used in contexts where geographical identification is necessary. They can be considered icons that are visually representative of a country or inter-governmental organisation. All flags are available in both circle and rectangle shapes.
```
Comment (joshwooding) [inline site/docs/foundations/iconography/country-symbols.mdx:6]: Do we want to use the term flags?

File: site/docs/foundations/iconography/country-symbols.mdx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+<Image
+  src="/img/foundations/iconography-countrysymbols-variant.png"
+  alt="Example of circular flag next to rectangular flag"
```
Comment (joshwooding) [inline site/docs/foundations/iconography/country-symbols.mdx:10]: Are they symbols or flags

Comment (amritadesmet) [reply site/docs/foundations/iconography/country-symbols.mdx:10]: Good point thanks, changed it!

**Review by amritadesmet — APPROVED** [review]

File: site/docs/foundations/iconography/country-symbols.mdx (hunk 4 above)
Comment (joshwooding) [reply site/docs/foundations/iconography/country-symbols.mdx:10]: @amritadesmet There are other references to flag

Comment (amritadesmet) [reply site/docs/foundations/iconography/country-symbols.mdx]: Changed the alt text.

**Review by amritadesmet — APPROVED** [review]

File: site/docs/foundations/iconography/country-symbols.mdx [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+title: Country symbols
+layout: DetailTechnical
+---
+
+Country symbols represent a specific country and are used in contexts where geographical identification is necessary. They can be considered icons that are visually representative of a country or inter-governmental organisation. All country symbols are available in both circle and rectangle shapes.
+
+<Image
+  src="/img/foundations/iconography-countrysymbols-variant.png"
+  alt="Example of circular country symbol next to rectangular country symbol"
```
Comment (joshwooding) [inline site/docs/foundations/iconography/country-symbols.mdx:10]:
```suggestion
  alt="Example of circular country symbols next to rectangular country symbols"
```

File: site/docs/foundations/iconography/country-symbols.mdx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+<ImageSwitcher
+  images={[
+    { src: "/img/foundations/scale-country.svg", alt: "" },
+    { src: "/img/foundations/scale-country-annotated.svg", alt: "" },
+  ]}
+  label="Show multipliers"
+/>
+
+| Size token         | Multiplier | High density (px) | Medium | Low | Touch |
```
Comment (joshwooding) [inline site/docs/foundations/iconography/country-symbols.mdx:29]: Is it an issue that High is the only one with the suffix and unit?

Comment (amritadesmet) [reply site/docs/foundations/iconography/country-symbols.mdx]: Edited it to match other table in foundations

**Review by amritadesmet — APPROVED** [review]

**Review by origami-z — APPROVED** [review]

**Review by amritadesmet — DISMISSED** [review]

**Review by joshwooding — APPROVED** [review]

**Review by amritadesmet — APPROVED** [review]

**Comment by joshwooding** [comment]
Merging this from now, but we can address any feedback in follow ups

**Pull request merged**